crates/subspace-runtime/Cargo.toml

```diff
@@ -73,9 +73,11 @@ default = ["std"]
 std = [
 	"codec/std",
 	"domain-runtime-primitives/std",
+	"frame-benchmarking?/std",
```

Comment:
What does question here mean?

Reply:
It means dependency will not be enabled due to this (it is optional), but if something else requires it, it will be enabled with this feature.

Reply:
I other words this is how you enable a feature for optional dependency without requiring dependency itself.